android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStorageModule.java

```diff
@@ -78,7 +80,7 @@ synchronized void scheduleNext() {
   private final SerialExecutor executor;
 
   public AsyncStorageModule(ReactApplicationContext reactContext) {
-    this(reactContext, AsyncTask.THREAD_POOL_EXECUTOR);
+    this(reactContext, BuildConfig.AsyncStorage_use_AsyncTaskThreadPoolExecutor?AsyncTask.THREAD_POOL_EXECUTOR:Executors.newSingleThreadExecutor());
```

Comment:
Nit: Reformat for better legibility.
```suggestion
    this(
      reactContext,
      BuildConfig.AsyncStorage_use_AsyncTaskThreadPoolExecutor
        ? AsyncTask.THREAD_POOL_EXECUTOR
        : Executors.newSingleThreadExecutor());
```